CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+Version 0.0.8
+-------------
+- [APP-3750](https://jira.nyt.net/browse/APP-3750) Update Purchase to include `autorenew` mock so can have sku type in Purchase
+
```

Comment:
This version is already on sonatype

https://mvnrepository.com/artifact/com.nytimes.android/register

Reply:
Bumped version to 0.0.9